app/views/match_decisions/thirteen/_accept_referral.haml

```diff
@@ -8,7 +8,7 @@
               .c-choice-icon.c-choice-icon--confirm.mr-4
               %h3 Accept Match
           .o-choose__content
-            = form.input :date_voucher_issued, label: 'Date voucher issued', as: :date_picker, disabled: !@decision.editable?
+            = form.input :date_voucher_issued, label: Translation.translate('Route Thirteen Date voucher issued'), as: :date_picker, disabled: !@decision.editable?
```

Comment:
Hmm, if we're going to add `Route Thirteen` we should probably provide a default translation to the previous version.  What do you think about just translating `Date voucher issued`?

Reply:
That makes sense to me, I almost went that way originally and it should be a simple update. I'll get it pushed to the PR here soon.

Reply:
This has been added in [this commit](https://github.com/greenriver/boston-cas/pull/886/commits/4ede0bbd22c83748f54321c4e4b22632d3a6373c)